ipywidgets/static/widgets/js/layout.js

```diff
@@ -36,18 +36,15 @@ define([
                 'align_items',
                 'align_self',
                 'bottom',
+                'border',
```

Comment:
@all it is not only about adding border, but about only using shorthand css properties when there is one in order to avoid the issues discussed in this issue. border is the top-most shorthand property covering the border widths, which concern layout in my opinion.